src/ui/qgsgraduatedsymbolrendererwidget.ui

```diff
@@ -6,15 +6,15 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>498</width>
-    <height>641</height>
+    <width>477</width>
```

Comment:
I wonder -- instead of adding the widgets directly in this (already very complex) widget, could we instead have a "..." or "settings" button next to the classification method combobox which opens a new pane allowing these method-specific widgets to be set?

Reply:
The UI is indeed crowded but I have the feeling it should not be the parameter of the method which should be hidden in a secondary UI.
Also, I would like to avoid extra dialog popping out, I love the flattening experience here (kudos to @NathanW2 for this awesome improvement).

From my experience, it's rather the legend I almost never touch (legend format, precision, trim trailing 0). Also, I always leave connecting classes checked.